valhalla/userrequests/duration_utils.py

```diff
@@ -42,14 +44,28 @@ def get_num_filter_changes(molecules):
     return len(list(itertools.groupby([mol.get('filter', '') for mol in molecules])))
 
 
-def get_molecule_duration_per_exposure(molecule_dict):
-    total_overhead_per_exp = configdb.get_exposure_overhead(molecule_dict['instrument_name'], molecule_dict['bin_x'])
+def get_molecule_duration_per_exposure(molecule_dict, window_dicts):
+    # The artificial delay for sinistro imaging readout will be removed on UT Feb 5 00:00. Use the real (lower) readout
+    # time of 26 seconds for sinistro imager requests whose earliest window starts after this date. This is a
+    # hack that will be removed after the transition date.
+    readout_switch_dt = datetime(year=2019, month=2, day=5, hour=0, minute=0, second=0, tzinfo=timezone.utc)
+
+    if window_dicts and sum([1 for w in window_dicts if 'start' in w]) > 0:
```

Comment:
Is this necessary? The Window serializer should set the window start to now if it isnt set

Reply:
I guess not. It can't hurt though and this is all going to be reverted.